src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -401,7 +401,7 @@ def add_electrodes(
         )
 
     # Find default values for custom (not schema defined) properties / columns already in the electrode table
-    type_to_default_value = {list: [], np.ndarray: np.array(np.nan), str: "", Real: np.nan}
+    type_to_default_value = {list: [], np.ndarray: np.array(np.nan), str: "", Real: np.nan, bool: False}
```

Comment:
What field is `bool` that requires this? Note that this type of change could potentially result in incorrect data, so we'd want to try to avoid it. Same reason we upcast `int` to `float` so we can designate `NaN` as 'unknown' rather than assuming some potentially incorrect default integer value

Reply:
Properties can be boolean (e.g., in our case we have a property `pass_qc` if the unit passes quality metrics). Currently, this is not supported (I believe it should though!)